src/models/class.js

```diff
@@ -0,0 +1,23 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+const passportLocalMongoose = require("passport-local-mongoose");
+
+const ClassSchema = new Schema({
```

Comment:
Good use of `required: true` to enforce that every class has a name. Consider adding additional validation if needed (ensuring unique class names, if applicable).